packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -2017,14 +2019,16 @@ export class UUIDV4 extends AbstractDataType<string> {
   }
 }
 
+export type IncludeAsCallback = (includeAs: string) => [Literal | Fn, string];
```

Comment:
I don't see a good reason to require the name of the attribute to be specified. We already know the name of the attribute. This function should only return the Expression, instead of a tuple